test/web-platform-tests/to-upstream/custom-elements/CustomElementRegistry-dont-upstream.html

```diff
@@ -0,0 +1,755 @@
+<!DOCTYPE html>
```

Comment:
If you wait for <https://github.com/jsdom/jsdom/pull/3139>, then this file becomes unnecessary.